internal/quickstart/container.go

```diff
@@ -18,21 +21,28 @@ const (
 	defaultEnvKey  = "test"
 )
 
+type step int
+
 const (
-	_ = iota
+	_ step = iota
 	stepCreateFlag
 	stepChooseSDK
 	stepShowSDKInstructions
 	stepToggleFlag
 )
 
+func (s step) String() string {
+	return [...]string{"_", "1 - feature flag name", "2 - sdk selection", "3 - sdk installation", "4 - flag toggle"}[s]
```

Comment:
Since this slice will always be small, we don't need to set the length of the backing array, so you could write this as
```
func (s step) String() string {
	return []string{
		"_",
		"1 - feature flag name",
		"2 - sdk selection",
		"3 - sdk installation",
		"4 - flag toggle",
	}[s]
}
```
for more readability.

Reply:
Sounds good